src/serverSetup/session.js

```diff
@@ -1,13 +1,11 @@
 import session from 'express-session'
 import { createClient } from 'redis'
 import RedisStore from 'connect-redis'
-import cookieParser from 'cookie-parser'
 import config from '../../config/index.js'
 import logger from '../utils/logger.js'
 import { types } from '../utils/logging.js'
 
 export async function setupSession (app) {
-  app.use(cookieParser())
```

Comment:
don't we need this?

Reply:
> Note Since version 1.5.0, the [cookie-parser middleware](https://www.npmjs.com/package/cookie-parser) no longer needs to be used for this module to work. This module now directly reads and writes cookies on req/res. Using cookie-parser may result in issues if the secret is not the same between this module and cookie-parser.

(from https://www.npmjs.com/package/express-session)